### PR TITLE
Revert "Revert "Gradle update""

### PR DESCRIPTION
**Pull request by wipfli**
Reverts maplibre/maplibre-gl-native#614

We want the gradle update which @artakka prepared, so let's introduce it again...

### CODE REVIEW

File: .github/workflows/android-ci-pull.yml [hunk 1]
```diff
@@ -63,6 +63,9 @@ jobs:
         with:
           distribution: 'zulu'
           java-version: '11'
+          
+      - name: Setup Android SDK for accepting tool licenses
+        uses: android-actions/setup-android@v2
```
Comment (wipfli) [inline .github/workflows/android-ci-pull.yml:69]: Maybe we should update the docker container instead...

File: .github/workflows/android-ci-pull.yml [hunk 2]
```diff
@@ -116,9 +119,17 @@ jobs:
...
       - name: Run Android unit tests
         run: make run-android-unit-test
-
+    
+      - name: Get latest CMake and Ninja
+        uses: lukka/get-cmake@latest
+        with:
+          cmakeVersion: 3.10.2
+          ninjaVersion: latest
+
```
Comment (wipfli) [inline .github/workflows/android-ci-pull.yml:128]: Similar here, we might want to update the docker container instead

File: .github/workflows/android-ci.yml [hunk 3]
```diff
@@ -63,6 +63,9 @@ jobs:
           distribution: 'zulu'
           java-version: '11'
 
+      - name: Setup Android SDK for accepting tool licenses
+        uses: android-actions/setup-android@v2
```
Comment (wipfli) [inline .github/workflows/android-ci.yml:67]: We might want to update the docker container instead...

**Comment by wipfli** [comment]
I started putting a bit of information together about the android docker build containers at https://github.com/maplibre/maplibre-gl-native/issues/622

**Review by birkskyum — APPROVED** [review]

**Comment by wipfli** [comment]
Feel free to merge this, @birkskyum. We can update the docker container in a follow-up pull request...

**Comment by artakka** [comment]
Let us have tasks as they are good changes to make in the future.

**Pull request merged**